schemas/vertices/chemical_reactions.json

```diff
@@ -0,0 +1,62 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "required": ["_key"],
+  "description": "Chemical reactions",
+  "properties": {
+    "_key": {
+      "type": "string",
+      "example": "rxn02201",
+      "description": "ms id",
+      "pattern": "^rxn\\d+$"
+    },
+    "direction": {
+      "type": "string",
+      "enum": [">", "<", "="]
+    },
+    "name": {
+      "type": "string",
+      "description": "Chemical names",
+      "example": "trans-2-Methyl-5-isopropylhexa-2,5-dienal dehydrogenase_c0"
+    },
+    "gpr": {
+      "type": "string",
+      "example": "PGN_RS01070"
+    },
+    "enzyme": {
```

Comment:
I think it would be better to be more explicit and call this ec_number in future

Reply:
I'll change the name. What does "ec" stand for so I can put it in the description?